core/src/main/scala/zio/config/ConfigStringModule.scala

```diff
@@ -244,7 +244,7 @@ trait ConfigStringModule extends ConfigModule with ConfigSourceStringModule {
     def file(path: String): ConfigDescriptor[File] = nested(path)(file)
 
     val float: ConfigDescriptor[Float] =
-      source(ConfigSource.empty, PropertyType.FloatType) ?? "value of type float"
+      sourceDesc(ConfigSource.empty, PropertyType.FloatType) ?? "value of type float"
```

Comment:
Why all the name changes, adding `Desc` as suffix?

Reply:
They are not exposed to users, but constructors that ensures lazyDesc is being used internally. Example nestedDesc ensures Nested is being called with lazyDesc. 

For these constructors, I thought of not using the same name as those functions exposed to the user. Example: “nested” that is available to the user through zio.config.ConfigDescriptor._

Another reason is, I couldn’t also use “lazy” because it’s a key word and I had to use lazyDesc. So to keep the consistency I thought of sufficing Desc to all constructor functions.